x/evm/keeper/state_transition.go

```diff
@@ -285,8 +285,9 @@ func (k *Keeper) ApplyTransaction(ctx sdk.Context, tx *ethtypes.Transaction) (*t
 		} else if commit != nil {
 			// PostTxProcessing is successful, commit the tmpCtx
 			commit()
-			// Since the post processing can alter the log, we need to update the result
+			// Since the post-processing can alter the log, we need to update the result
 			res.Logs = types.NewLogsFromEth(receipt.Logs)
+			// TODO: figure out how to make this work with typed events
```

Comment:
No need to do anything here since the `EmitTypedEvent` calls the `EmitEvent` function after the typed event is converted to an `abci.Event`